docs/configuration/configuration.md

```diff
@@ -135,6 +135,18 @@ job_name: <job_name>
 # when a time series does not have a given label yet and are ignored otherwise.
 [ honor_labels: <boolean> | default = false ]
 
+# honor_timestamps controls how prometheus respects the timestamps exposed by
+# the target, providing the ability to discard timestamps. While timestamps
+# are most of the time desirable, some edge cases might require you to discard
```

Comment:
This is misleading, the timestamps on exposition are generally not desirable.

Reply:
I rephrased.